### PR TITLE
Fix PolynomialLR power type.

### PR DESCRIPTION
**Pull request by hiyuh**
`torch.optim.lr_scheduler.PolynomialLR` `power` was typed `int`, but should be `double`.
Non-integer `power` is widely used for common training recipe.
E.g. `torchvision`'s pre-trained semantic segmentation models uses `PolynomialLR` as main LR scheduler with `power = 0.9`; https://github.com/pytorch/vision/blob/main/references/segmentation/train.py#L201
See also https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.PolynomialLR.html

### CODE REVIEW

**Comment by hiyuh** [comment]
@dotnet-policy-service agree company="HACARUS"

**Comment by hiyuh** [comment]
regarding `TorchSharp.TestJIT.TestLoadJIT_3` failed,
i have no idea for now, since this PR would not intend to change any exception handling.

**Review by ds5678 — APPROVED** [review]

**Review by alinpahontu2912 — CHANGES_REQUESTED** [review]
Hey, LGTM, thanks for the fix :). Can you update your branch and also add the some notes in Releasenotes ?

**Comment by alinpahontu2912** [comment]
> regarding `TorchSharp.TestJIT.TestLoadJIT_3` failed, i have no idea for now, since this PR would not intend to change any exception handling.

It is a flaky test, we will try running again, don't worry

**Comment by hiyuh** [comment]
@alinpahontu2912 
* added `RELEASENOTES.md` update after rebase.
  + according to `CONTRIBUTING.md`, rebase is preferred, instead of merge commit.
  + however, it looks you're doing a merge commit in your branch.
    - https://github.com/alinpahontu2912/TorchSharp/commit/e7e07ea3fe486701d489d84f28b08085f0f09be2
  + i'm not sure which is true preference.

File: RELEASENOTES.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Releases, starting with 9/2/2021, are listed with the most recent release at the
 __Bug Fixes__:
 
 #1426 Sequential.eval() does not put model into eval mode<br/>
+#1440 `torch.optim.lr_scheduler.PolynomialLR` `power` type has been corrected, is now double.<br/>
```
Comment (alinpahontu2912) [inline RELEASENOTES.md:9]: Hey, just like in the other PR, the number should point to an issue or discussion thread and since this is not one, I say we follow the pattern from here: https://github.com/dotnet/TorchSharp/blob/5ba14022749a20b98798105a32988da59f2356ca/RELEASENOTES.md?plain=1#L287

Comment (hiyuh) [reply RELEASENOTES.md]: thanks, removed the number.
feel free to mark resolved if ok.

**Review by alinpahontu2912 — DISMISSED** [review]
Awesome, thanks 👍

**Comment by alinpahontu2912** [comment]
Hey @hiyuh, one of your PRs got merged, thanks for the contribution! Can you fix the Releasenotes conflict so we can merge this one too ?

**Comment by hiyuh** [comment]
@alinpahontu2912 
done & go ahead. :)

**Review by alinpahontu2912 — APPROVED** [review]
